webapp/src/components/Header/Header.tsx

```diff
@@ -23,7 +22,7 @@ import { useHistory } from "react-router-dom";
 import { useLendLink } from "shared/lib/hooks/useLendLink";
 import { BuyButton } from "../Wallet/BuyButton";
 
-const HeaderContainer = styled.div<MobileMenuOpenProps>`
+export const HeaderContainer = styled.div<MobileMenuOpenProps>`
```

Comment:
These shouldn't be imported directly from `webapp`, best if we migrate these components into `shared`